docs/detections/rules-ui-create.asciidoc

```diff
@@ -560,9 +567,13 @@ used as an identifier across systems
 * `{{rule.type}}`: Type of rule
 * `{{state.signals_count}}`: Number of alerts detected
 
-NOTE: The `{{context.rule.severity}}` and `{{context.rule.risk_score}}`
-placeholders contain the rule's default values even when the *Severity override*
-and *Risk score override* options are used.
+The following variables can only be passed if the rule’s action frequency is per alert: 
```

Comment:
To match the setting name in the UI (debatable whether to cap & bold too, I think it's OK not to):
```suggestion
The following variables can only be passed if the rule’s action frequency is for each alert: 
```